custom_model_runner/datarobot_drum/runtime_parameters/runtime_parameters.py

```diff
@@ -42,3 +50,73 @@ def get(key):
             )
 
         return transformed_env_value["payload"]
+
+    @classmethod
+    def mangled_param_name(cls, param_name):
```

Comment:
what is the purpose of the mangled param name?  why is it "mangled"?

Reply:
```suggestion
    def namespaced_param_name(cls, param_name):
```
It's not really _mangled_ it is just namespaced/prefixed (so users don't collide with other env vars DataRobot may set)